protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/generation/GenerationRequest.java

```diff
@@ -59,6 +61,9 @@ public interface GenerationRequest {
 
   Path getOutputDirectory();
 
+  @Nullable
+  File getDescriptorFile();
+
```

Comment:
Do we want to call this `outputDescriptorFile`, both here and in the AbstractGenerateMojo, just in case we want to have input descriptor files eventually as well?

Reply:
Renamed